pkg/cmd/pipeline/start_test.go

```diff
@@ -117,7 +117,7 @@ func Test_start_pipeline_not_found(t *testing.T) {
 
 	pipeline := Command(p)
 	got, _ := test.ExecuteCommand(pipeline, "start", "test-pipeline-2", "-n", "ns")
-	expected := "Error: " + errInvalidPipeline.Error() + "\n"
+	expected := "Error: " + fmt.Sprintf(errInvalidPipeline, "test-pipeline-2", "ns") + "\n"
```

Comment:
its better to use exact message, not fmt.Sprintf, you may have test failed for the typo here if dont this way

Reply:
heh, indeed